### DISCUSSION TITLE
Acquire a js.org subdomain

### DISCUSSION THREAD

**Discussion by aleclarson** [radashi-org/radashi#183] [Ideas]
https://github.com/js-org/js.org

**Comment by MarlonPassos-git** [comment]
Wow, that's cool. I didn't know this project existed. I was already thinking about doing a crowdfunding campaign to get a domain.

I'll take a look, if anything happens I'll make a PR.

**Comment by MarlonPassos-git** [comment]
- [x]  Config the CNAME file in [Radashi documentation repository](https://github.com/radashi-org/radashi-org.github.io)
- [x]  Create PR at [jr.org repository](https://github.com/js-org/js.org/tree/master)
- [x] When the URL lives, should update the JSDocs in [Radashi repository](https://github.com/radashi-org/radashi)

**Reply by MarlonPassos-git** [reply]
https://github.com/radashi-org/radashi-org.github.io/pull/1

**Reply by MarlonPassos-git** [reply]
https://github.com/js-org/js.org/pull/9206

**Reply by MarlonPassos-git** [reply]
https://github.com/radashi-org/radashi/pull/206

**Reply by MarlonPassos-git** [reply]
https://github.com/radashi-org/radashi-org.github.io/pull/2

**Reply by aleclarson** [reply]
I wonder if renaming the `radashi-org.github.io` repo to `radashi.js.org` is possible, or if that would break it somehow. 🤔

**Reply by MarlonPassos-git** [reply]
If you do not change the URL in the github pages. Works. But if the URL changes, we need to update the PR in [js-org](https://github.com/js-org/js.org/tree/master)

**Comment by MarlonPassos-git** [comment]
All PRs merged. 

The new domain is already working 🥳🥳🥳

https://radashi.js.org
